src/ScreenshotWindow.vala

```diff
@@ -48,7 +48,7 @@ namespace Screenshot {
                 border_width: 6,
                 deletable: false,
                 resizable: false,
-                title: _("Screenshot")
+                title: null
```

Comment:
Is there a reason to explicitly set this to null versus just omitting?

Reply:
Nope, I'm dumb :p